foundation/foundation-mda/src/main/resources/templates/deployment/configuration-store/v2/configuration-store.values.yaml.vm

```diff
@@ -21,7 +21,6 @@ aissemble-configuration-store-chart:
       supplementalVolumeMounts:
         - name: configurations
           mountPath: "/configurations"
-          readOnly: true
```

Comment:
q: do we not we want to keep the original PVC as read only?

Reply:
Ideally yes, but it introduces a bug with argoCD (#309). So this is a temp fix for MVP until we get time to further investigate the issue.